frontend/src/components/LandingSearchTable.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <section>
+        <b-table
+            :data="tableData"
+            detailed
+            detail-key="id">
+            <template slot-scope="props">
+                <b-table-column label="№ АСВП" width="50">
+                    {{props.row.id}}
+                </b-table-column>
+                <b-table-column label="Facility" width="50">
+                    {{props.row.facility}}
+                </b-table-column>
+                <b-table-column label="Дата відкриття" width="50">
+                    {{props.row.date.toLocaleDateString('uk-UA', dateViewOptions)}}
+                </b-table-column>
+                <b-table-column label="Стан" width="50">
+                    {{props.row.state}}
+                </b-table-column>
+            </template>
+            <template slot="detail" slot-scope="props">
+                Тут повинна бути інформація про боржників та стягувачів для впровадження номер {{props.row.id}}
+            </template>
+        </b-table>
+    </section>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class LandingSearch extends Vue {
+  public data(): object {
```

Comment:
Replace `object` with real type

Reply:
@k-gregory What type do you mean exactly? Should I introduce the type for current component's data context?

Reply:
Yes, I believe such typecheck will be useful when data starts coming from network.